<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Alias</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>for:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="beforeText"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>use alias:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="afterText"/>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="autoRadio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Keep episode number and season number for all episodes</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="manualRadio">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure custom season/episode rules:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>521</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Configured Custom Season Rules</string>
   </property>
   <widget class="QListWidget" name="ruleList">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>411</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>98</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveUpButton">
       <property name="text">
        <string>Move Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveDownButton">
       <property name="text">
        <string>Move Down</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>87</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>480</y>
     <width>87</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>531</width>
     <height>134</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>72</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Custom Season Rule</string>
    </attribute>
    <widget class="QPushButton" name="addButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>87</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add rule</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="resetEpisodes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset episode numbers in this range</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Use Season number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="seasonEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>for:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="allOrRangeCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>All episodes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Episodes starting from</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fromEpisodeEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="toOrOnwardsCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>onwards</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>to</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="toEpisodeEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>34</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Custom Episode Rule</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="addMinusCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Add</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Minus</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="episodeAmountEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>To/From Episode Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>for:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Season</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="episodeSeasonEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QComboBox" name="episodeAllOrRangeCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>All episodes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Episodes starting from</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="episodeFromEpisodeEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="episodeToOrOnwardsCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>onwards</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>to</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="episodeToEpisodeEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="episodeAddButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add rule</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
